src/pages/User/index.vue

```diff
@@ -2,10 +2,4 @@
   <div>User Page</div>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
-
-export default defineComponent({
-  name: 'User',
-});
-</script>
+<script setup lang="ts"></script>
```

Comment:
Is this really needed, for example using lang

Reply:
I think that lang isn't needed, but I'm not sure about the script, since it has the basic configuration for the file (I think that when you use setup at the script tag, Vue automatically adds the basic configuration for the component). I will try removing it, if not works I will just remove the lang

Reply:
I confirmed that is not needed, thanks for the suggestion